### PR TITLE
fix: token expiry for GlobalMethod

### PR DESCRIPTION
**Pull request by jptissot**
`createWorkflowToken` now defaults to 36500 if no value or 0 is specified

### CODE REVIEW

**Comment by sebastienros** [comment]
Can you document that method, couldn't find it in the workflows documentation

**Comment by jptissot** [comment]
Do we have a central spot to document scripts methods ?

**Comment by jptissot** [comment]
Maybe I could start something here : https://orchardcore.readthedocs.io/en/dev/docs/reference/modules/Scripting/

File: src/docs/reference/modules/Scripting/README.md [hunk 1]
```diff
@@ -52,3 +56,81 @@ The File scripting engine provides methods to read file contents.
...
+#### Generic functions
+
+| Function | Description 
+| -------- | ----------- |
+| `log(level: String, text: String, param: Object): void` | Formats and writes a log message at the specified log level. |
+| `uuid(): String` | Generates a unique identifier for a content item. |
+| `base64(String): String` | Decodes the specified string from Base64 encoding. Use https://www.base64-image.de/ to convert your files to base64. |
+| `html(String): String` | Decodes the specified string from HTML encoding. |
+| `gzip(String): String` | Decodes the specified string from gzip/base64 encoding. Use http://www.txtwizard.net/compression to gzip your strings. |
+| `variables()` | TODO |
```
Comment (jptissot) [inline src/docs/reference/modules/Scripting/README.md]: @agriffard If you have time to have a look.

File: src/docs/reference/modules/Scripting/README.md [hunk 2]
```diff
@@ -52,3 +56,81 @@ The File scripting engine provides methods to read file contents.
...
+| `queryString(name: String): String | Array` | Returns the entire query string (including the leading `?`) when invoked with no arguments, or the value(s) of the parameter name passed in as an argument. |
+| `responseWrite(text: String): void` | Writes the argument string directly to the HTTP response stream. |
+| `absoluteUrl(relativePath: String): String` | Returns the absolute URL for the relative path argument.  |
+| `readBody(): String` | Returns the raw HTTP request body.  |
+| `requestForm(name: String): String | Array` | Returns the value(s) of the form field name passed in as an argument. |
+| `queryStringAsJson(): JObject` | Returns the entire query string as a JSON object. Example: ` { "param1": [ "param1-value1", "param1-value2" ], "param2": [ "param2-value1", "param2-value2" ], ... }` |
+| `requestFormAsJson(): JObject` | Returns the entire request form as a JSON object. Example: ` { "field1": [ "field1-value1", "field1-value2" ], "field2": [ "field2-value1", "field2-value2" ], ... }` |
+
+
+#### Workflows (`OrchardCore.Workflows.Http`)
```
Comment (jptissot) [inline src/docs/reference/modules/Scripting/README.md]: I moved and reformatted these from the workflow module. We should probably link to these from the workflow docs page.

**Pull request merged**